web/src/components/RemoteContent.tsx

```diff
@@ -145,10 +149,25 @@ const RemoteContent = ({
         node.addEventListener('click', handleClick)
       }
     })
+
+    // iframe handling - remove not allowed iframe src from DOM and add tracking parameter
+    const iframes = sandBoxRef.current.getElementsByTagName('iframe')
+    Array.from(iframes).forEach((node: HTMLIFrameElement) => {
+      if (buildConfig().allowedIframeSources.some(el => node.src.indexOf(el) > 0) && sandBoxRef.current) {
```

Comment:
You already using `sandBoxRef.current` above without a non-null check, so it should be fine here as well.
```suggestion
      if (buildConfig().allowedIframeSources.some(source => node.src.indexOf(source) > 0)) {
```